generators/app/templates/app/controllers/v1/Auth.ts

```diff
@@ -44,42 +56,343 @@ export interface JWTPayload {
   email: string;
   role: User["role"];
 }
+@Controller("auth")
+// @ApiDocs()
```

Comment:
Why is this commented?

Reply:
> Why is this commented?
@rodksquare 
The auth controller doesn't have a model, so its requests and responses are not fully defined, so its documentation is optional or all requests and responses must be manually defined.

Reply:
can we delete that line?

Reply:
yes we can , I will delete it.